tests/e2e/performance/csi_tests/test_bulk_pod_attachtime_performance.py

```diff
@@ -78,19 +67,15 @@ def base_setup(self, project_factory, interface_type, storageclass_factory):
         argvalues=[
             pytest.param(
                 *[constants.CEPHBLOCKPOOL, 120],
-                marks=[pytest.mark.performance],
             ),
             pytest.param(
                 *[constants.CEPHBLOCKPOOL, 240],
-                marks=[pytest.mark.performance],
             ),
             pytest.param(
                 *[constants.CEPHFILESYSTEM, 120],
-                marks=[pytest.mark.performance],
             ),
             pytest.param(
                 *[constants.CEPHFILESYSTEM, 240],
-                marks=[pytest.mark.performance],
```

Comment:
Avi, why did you remove the mark ?

Reply:
since we have a global performance marker on the class level, so we don't need this marker per test.